### PR TITLE
Delete Initiative

### PR DESCRIPTION
**Pull request by kevinyu05**

### CODE REVIEW

File: spec/features/initiative_spec.rb [hunk 1]
```diff
@@ -40,6 +40,12 @@
     expect(page).to have_content('TestB')
     expect(page).to_not have_content('TestA')
   end
+  scenario 'delete initiative from list view' do
+    visit root_path
+    expect{@initiative.destroy}.to change(Initiative,:count).by(-1)
+    expect(page).to have_content('Initiatives')
```
Comment (jasononaquest) [inline spec/features/initiative_spec.rb]: What kind of test could we do here to make sure the deleted initiative is _not_ on the page?

File: app/views/initiatives/index.html.erb [hunk 2]
```diff
@@ -4,18 +4,26 @@
...
+          <h3 class="init-name"><%= initiative.title %></h3>
+          <p class="init-description"><%= initiative.description %></p>
+          <div>
+            <div class="init-footer w-row">
+              <div class="w-col w-col-2" style="padding: 0;"><%= image_tag "icon-calendar.svg", class: "icon-target" %></div>
+              <div class="card-date w-col w-col-10">
+                <%= initiative.target_date.to_s(:long_ordinal) unless initiative.target_date.nil? %>
+              </div>
+              <div class="w-col w-col-2" style="padding: 0;">
+                <%= link_to initiative_path(initiative), method: :delete, data: { confirm: 'Are you sure?' }, :style=> "text-decoration: none;" do %>
```
Comment (jasononaquest) [inline app/views/initiatives/index.html.erb]: What can we do to make sure the user knows which initiative they're  deleting?

Screen Shot 2019-03-14 at 11 23 29 AM

File: spec/features/initiative_spec.rb (hunk 1 above)
Comment (kevinyu05) [reply spec/features/initiative_spec.rb]: We updated the feature test to ensure initiative deletion is not shown on the page

File: app/views/initiatives/index.html.erb (hunk 2 above)
Comment (kevinyu05) [reply app/views/initiatives/index.html.erb]: We altered the confirmation message to exhibit which initiative the user is deleting

**Pull request merged**